fenix/app/src/androidTest/java/org/mozilla/fenix/ui/CreditCardAutofillTest.kt

```diff
@@ -0,0 +1,263 @@
+package org.mozilla.fenix.ui
+
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.fenix.customannotations.SmokeTest
+import org.mozilla.fenix.helpers.AndroidAssetDispatcher
+import org.mozilla.fenix.helpers.HomeActivityIntentTestRule
+import org.mozilla.fenix.helpers.TestAssetHelper
+import org.mozilla.fenix.helpers.TestHelper.exitMenu
+import org.mozilla.fenix.ui.robots.homeScreen
+import org.mozilla.fenix.ui.robots.navigationToolbar
+import java.time.LocalDate
+
+class CreditCardAutofillTest {
+    private lateinit var mockWebServer: MockWebServer
+
+    object FirstCreditCard {
```

Comment:
Maybe `MockCreditCard1` and `MockCreditCard2`

Reply:
Done !